### ISSUE TITLE
On-prem TFS 2018 Support

### ISSUE THREAD

**Issue by dukesta3** [jgarverick/RequirementsIntegrator#6]
My team just moved over from Jira to on-prem TFS 2018.  This project seems like it could really help us solve a requirements traceability gap with TFS.  Does this work with TFS 2018?

**Comment by jgarverick** [comment]
I haven’t tested it with 2018 to be honest...if you install it and it doesn’t work, please let me know and I will make every attempt to get that resolved.

**Comment by adammulroy** [comment]
@jgarverick I installed this extension on a on-prem version of TFS 2018. Initially, I couldn't get it installed. I looked through the documentation for VSO Extensions, and saw they've updated their targets.

I made the following change in your vss-extension.json:
```
  "targets": [
    {
      "id": "Microsoft.VisualStudio.Services"
    }
```

I was able to install the extension with the above change. However, It has some bugs. Once I've uploaded requirements, and attempt to map them to a PBI or Bug, I get the following error:
![pbi_error](https://user-images.githubusercontent.com/6828663/30763978-8f851a9e-9fae-11e7-97b1-459026a46021.png)

Additionally, when I upload requirements they seem to disappear after a few minutes.

I think this extension would suit or needs and provided "just enough" traceability for our requirements without slowing us down. 

Let me know if you need any further info to assist with fix actions. Thanks!

**Comment by jgarverick** [comment]
Thanks...I’m sure the issue is the version of the SDK that’s referenced. I’ll see what I can do about getting things updated soon. Hope this helps!

**Comment by jgarverick** [comment]
I've just published version 2.0.1, which offers on-prem and VSTS support.  Please give it a spin and let me know if you hit any snags!
